src/views/joinStaff/joinStaff.jsx

```diff
@@ -15,32 +15,39 @@ export const JoinStaff = () => {
   const auth_headers = { headers: { 'Authorization': `Bearer ${useContext(UserContext).user.accessJwt}` } };
 
   useEffect(() => {
-    const URLs = ['/api/user?r=3', '/api/user?r=4'];
-    const fetchData = URL => {
-      return axios
-        .get(URL, auth_headers)
+    const fetchData = () => {
+      axios
+        .get('/api/user?r=3', auth_headers)
         .then(res => {
-          setStaff(...staff, res.data.users);
+          var joinStaff = res.data.users;
+          axios
+            .get('/api/user?r=4', auth_headers)
+            .then(res2 => {
+              var admins = res2.data.users;
+
+              setStaff(joinStaff.concat(admins));
+            })
         })
         .catch(error => {
           Toast(error.message, 'error');
-          console.log(error);
         });
     };
-    Promise.all(URLs.map(url => fetchData(url)));
+    fetchData();
   }, []);
 
   const staffCard = (user) => {
-    return (
-      <JoinStaffCard
-        key={user.id}
-        name={`${user.firstName} ${user.lastName}`}
-        phoneNumber={user.phone}
-        email={user.email}
-        tickets={user.tickets}
-        tenants={user.tenants}
-        admin={user.role == RoleEnum.ADMIN} />
-    );
+    return user ?
+      (
+        <JoinStaffCard
+          key={user.id}
+          name={`${user.firstName} ${user.lastName}`}
+          phoneNumber={user.phone}
+          email={user.email}
+          tickets={user.tickets}
+          tenants={user.tenants}
```

Comment:
I'm getting console errors when viewing the `/staff` endpoint. 
```
Invalid prop `tenants` of type `string` supplied to `JoinStaffCard`, expected `number`.
    in JoinStaffCard (at joinStaff.jsx:41)
```

Reply:
I'll fix this, thanks for finding it!

Reply:
So looks like our backend is missing some info for the user, so I'll remove the props for this for now
![image](https://user-images.githubusercontent.com/8600792/120579747-e826a400-c3dc-11eb-9bc6-00f35ef862f2.png)